### PR TITLE
Aleutian West admin data fixes

### LINKED ISSUE #215: Aleutians West has conflicting admin data
https://whosonfirst.mapzen.com/spelunker/id/102080627/

Hierarchy says it belongs to Ontario, Canada, but QS fields say Alaska, US.

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #215. 

After poking around with the QS imports, I noticed there was some weirdness in Ontario... led me to realize that it was due to the international date line and that we already had an issue open. This PR fixes the **admin** records of Aleutian West and all descendants, though it does not include `campus` and `postalcode` record updates (needs to be done in another PR). Includes the following for **admin** records:
- Updated hierarchies to include Alaska and the US instead of Canada and Ontario 
- New `parent_id` for Alaska
- Updated `wof:belongsto` to reflect new hierarchy
- Added in `wof:country` of "US" whenever needed
- Exportified to update `lastmodified` field

### CODE REVIEW

**Comment by stepps00** [comment]
@nvkelso - success! 

This PR should take care of **admin** records in Alaska that were incorrectly parented to Ontario (intl dateline strikes again).

File: data/102/080/627/102080627.geojson [hunk 1]
```diff
@@ -41,15 +40,14 @@
     "wof:hierarchy":[
         {
             "continent_id":102191575,
-            "country_id":85633041,
-            "county_id":102080627,
-            "region_id":85682057
```
Comment (nvkelso) [inline data/102/080/627/102080627.geojson]: Needs to have the Alaska `region_id`.

**Comment by nvkelso** [comment]
Besides fixing the region in https://github.com/whosonfirst-data/whosonfirst-data/pull/357/files#r76685185, LGTM. 

Will require re-PIPing venues.

File: data/102/080/627/102080627.geojson (hunk 1 above)
Comment (stepps00) [inline data/102/080/627/102080627.geojson]: Record now has the Alaska `region_id` (85688573).

**Comment by stepps00** [comment]
Removed merge conflict from PR and added the `region_id` to 102080627.geojson
- over to @thisisaaronland for review.

**Comment by nvkelso** [comment]
Back to @stepps00 to fix merge conflict.

**Comment by stepps00** [comment]
Most recent commit (23e9aa7) fixes merge conflict.

**Comment by stepps00** [comment]
Most recent commit adds `reversegeo:*` properties to the Aleutian West record and PIPs descendant records to update hierarchies.

**Comment by stepps00** [comment]
@nvkelso - OK to merge? PIP work done on descendants, should be good to merge now.

**Pull request merged**